src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -350,15 +379,17 @@ void FancyZones::ToggleEditor() noexcept
         DPIAware::GetScreenDPIForWindow(foregroundWindow, dpi_x, dpi_y);
     }
 
+    JSONHelpers::FancyZonesDataInstance().CustomZoneSetsToJsonFile(iter->second->GetCustomZoneSetsTmpPath());

```

Comment:
`iter->second` is used a lot here, could we refactor it to a variable with a descriptive name?